templates/index.html

```diff
@@ -1,692 +1,425 @@
+        <div class="p-navigation__banner">
+          <div class="p-navigation__logo">
+            <a href="/">
+              <img class="u-float-left" src="https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg" width="148" alt="Snapcraft.io" />
+            </a>
+          </div>
+          <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+          <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
         </div>
-    </section>
-    <section id="snapcraft_home_install" class="row hero-row no-border row-grey">
-        <div class="wrapper">
-            <div class="twelve-col">
-                <div class="equal-height--vertical-divider">
-                  <div class="equal-height--vertical-divider__item four-col">
-                      <h3>Quick to install</h3>
-                      <p>Snaps are quick to install, easy to create, safe to run, and they
-                      update automatically and transactionally so your app is always fresh and
-                      never broken.</p>
-                      <p><a href="/docs/core/install">Install snapd&nbsp;&rsaquo;</a></p>
-                  </div>
-                  <div class="equal-height--vertical-divider__item four-col">
-                      <h3>Work anywhere</h3>
-                      <p>Snaps aim to work on any distribution or device, from IoT devices to
-                      servers, desktops to mobile devices.</p>
-                      <p><a href="https://github.com/snapcore/snapd" class="external">Build snapd from source</a></p>
-                  </div>
-                  <div class="equal-height--vertical-divider__item four-col last-col">
-                      <h3>GitHub and beyond</h3>
-                      <p>The public collection of snaps includes the best of GitHub and beyond,
-                      so you have the whole world of Linux at your fingertips.</p>
-                      <p><a href="https://build.snapcraft.io" class="external">Build a snap from GitHub</a></p>
-                  </div>
-                </div>
-            </div>
+        <nav class="p-navigation__nav" role="menubar">
+          <span class="u-off-screen">
+            <a href="#main-content">Jump to main content</a>
+          </span>
+          <ul class="p-navigation__links" role="menu">
+            <li class="p-navigation__link" role="menuitem"><a href="https://build.snapcraft.io">Build</a></li>
+            <li class="p-navigation__link" role="menuitem"><a href="/docs">Docs</a></li>
+            <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="https://tutorials.ubuntu.com">Tutorials</a></li>
+            <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="https://forum.snapcraft.io/categories">Forum</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="p-strip--accent is-deep snapcraft-hero">
```

Comment:
Please add `id="main-content"` here please. So the Jump to main content link above has a target.